apps/src/client.rs

```diff
@@ -485,80 +469,75 @@ pub fn connect(
             scid_sent = true;
         }
 
-        if args.perform_migration &&
+        if conn_args.multipath &&
```

Comment:
When the number of addresses to use is greater than conn.available_dcids() some addresses are ignored without giving any information. I think a warning could be added to show that this happens and it can be solved with the --max-active-cids parameter

Reply:
It might be indeed interesting to add a warning message indicating so, good point!